query-engine/driver-adapters/src/proxy.rs

```diff
@@ -261,8 +275,18 @@ fn js_value_to_quaint(
         },
         ColumnType::Bytes => match json_value {
             serde_json::Value::String(s) => QuaintValue::Bytes(Some(s.into_bytes().into())),
+            serde_json::Value::Array(array) => {
```

Comment:
TODO for future: unify these two branches, get rid of JSON arrays here and use base64-encoded strings everywhere (in results of the queries, not in parameters)